presto-docs/src/main/sphinx/release/release-347.md

```diff
@@ -0,0 +1,55 @@
+# Release 347 (XX Nov 2020)
+
+## General Changes
+
+* Add `ALTER VIEW ... SET AUTHORIZATION` syntax to allow changing the view owner. ({issue}`5789`)
+* Add support for `ALL` qualifier for `INTERSECT` and `EXCEPT`. ({issue}`2152`)
+* Add {func}`contains_sequence` function. ({issue}`5593`)
+* Add support for configuring subnet based network topology for topology-aware scheduling. ({issue}`4862`) 
+* Improve query performance by reducing worker to worker communication overhead. ({issue}`5905`, {issue}`5949`)
+* Improve result throughput over fast network links by making http response compression configurable. Clients
+  can disable compressed responses using the `--disable-compression` flag or `disableCompression` driver property.
+  Compression can be disabled server-wide with `query-results.compression-enabled=false`. ({issue}`5818`)
+* Rename ``rewrite-filtering-semi-join-to-inner-join`` session property to ``rewrite_filtering_semi_join_to_inner_join``. ({issue}`5954`)
+* Throw a user error for failing to decode a session property value. ({issue}`5731`)
+* Fix failure when expressions that produce values of type `row` are used in a `VALUES` clause. ({issue}`3398`)
+
+## Security Changes
+
+* Add support for multiple LDAP bind patterns. ({issue}`5874`)
+
+## Web UI
+
+## JDBC Driver Changes
+
+* Implement `addBatch()`, `clearBatch()` and `executeBatch()` methods in `PreparedStatement`. ({issue}`5507`)
+
+## CLI Changes
+
+* presto-cli now accepts queries passed via shell redirection. ({issue}`5881`)
+
+## Docker Image Changes
+
+* Update Presto docker image to use Centos 8 as the base. ({issue}`5920`)
+
+## Hive Connector Changes
+
+* Add support for `ALTER VIEW  ... SET AUTHORIZATION` SQL syntax to change the view owner. This supports Presto and Hive views. ({issue}`5789`)
+* Allow configuring HDFS replication factor through property `hive.dfs.replication`. ({issue}`1829`)
+* Add access checks for tables in Hive Procedures. ({issue}`1489`)
```

Comment:
This is a security bug fix, not "a feature" as it sounds now

Reply:
It is. As far as I know, it has two parts: 
* Add support in the SPI
* Enforce access control checks within Hive procedures